src/config/mod.rs

```diff
@@ -363,7 +364,7 @@ impl Display for Resource {
         match self {
             Resource::Port(address, protocol) => write!(fmt, "{} {}", protocol, address),
             Resource::SystemFdOffset(offset) => write!(fmt, "systemd {}th socket", offset + 1),
-            Resource::Stdin => write!(fmt, "stdin"),
+            Resource::Fd(fd) => write!(fmt, "fd: {}", fd),
```

Comment:
I think we should expand this acronym:
```suggestion
            Resource::Fd(fd) => write!(fmt, "file descriptor #{}", fd),
```
This will, of course, change all the expected error texts below.